turtlebot3c-bringup-snap/snap/local/update_firmware.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+arch=$(uname -m)
+
+if [ "${arch}" = "x86_64" ]; then
```

Comment:
I understand that this is the one option out of three that's not supported but shouldn't we test for 'not arm' instead? That would be proper.

Reply:
The snaps build for `arm64`, `armhf` and `x86_64`.  With the `uname` cmd we get the architecture but we don't get the generic ARM or x86. I found it simpler to look for `x86_64` rather looking for the two arm architectures.